src/hotspot/cpu/aarch64/gc/g1/g1BarrierSetAssembler_aarch64.cpp

```diff
@@ -274,6 +275,7 @@ void G1BarrierSetAssembler::load_at(MacroAssembler* masm, DecoratorSet decorator
                          dst /* pre_val */,
                          rthread /* thread */,
                          tmp1 /* tmp */,
```

Comment:
For tmp1 and tmp2 the comments, to match the paramter names, would also be "/* tmp1 */" and "/* tmp2 */".

Reply:
Fixed 👍